.circleci/config.yml

```diff
@@ -445,67 +423,62 @@ workflows:
             - unit-test-casskop
           <<: *canonical_repo_filter
 
-      - minikube/e2e-orb:
+      - k3d/e2e-orb:
           name: e2e-test RollingRestart
           test_name: RollingRestart
           requires:
-            - unit-test-casskop
+            - build-casskop
           <<: *ignore_fork_pr_filter
 
-      - minikube/e2e-orb:
+      - k3d/e2e-orb:
           name: e2e-test ScaleUp
           test_name: ClusterScaleUp
           requires:
-            - unit-test-casskop
+            - build-casskop
           <<: *ignore_fork_pr_filter
 
-      - minikube/e2e-orb:
+      - k3d/e2e-orb:
           name: e2e-test ScaleDown
           test_name: ClusterScaleDown
           requires:
-            - unit-test-casskop
+            - build-casskop
           <<: *ignore_fork_pr_filter
 
-      - minikube/e2e-orb:
+      - k3d/e2e-orb:
           name: e2e-test ScaleDownSimple
           test_name: ClusterScaleDownSimple
           requires:
-            - unit-test-casskop
+            - build-casskop
           <<: *ignore_fork_pr_filter
 
-      - minikube/e2e-orb:
+      - k3d/e2e-orb:
           name: e2e-test CreateOneClusterService
           test_name: CreateOneClusterService
           requires:
-            - unit-test-casskop
+            - build-casskop
           <<: *ignore_fork_pr_filter
 
-      - minikube/e2e-orb:
+      - k3d/e2e-orb:
           name: e2e-test UpdateConfigMap
           test_name: UpdateConfigMap
           requires:
-            - unit-test-casskop
+            - build-casskop
           <<: *ignore_fork_pr_filter
 
-      - minikube/e2e-orb:
+      - k3d/e2e-orb:
           name: e2e-test ExecuteCleanup
           test_name: ExecuteCleanup
           requires:
-            - unit-test-casskop
+            - build-casskop
           <<: *ignore_fork_pr_filter
+
       - operator/publish-helm:
           name: publish-helm-orange-incubator-casskop
           requires:
             - report-casskop
           <<: *only_on_release_tag
-      - deploy-website:
```

Comment:
Why do you remove the website publication ?

Reply:
I’ll fix that one